README.md

```diff
@@ -123,7 +123,8 @@ Make sure to clone this repository in your `GOPATH` - otherwise compilation fail
 
 ```sh
 $ git clone https://github.com/projectatomic/skopeo $GOPATH/src/github.com/projectatomic/skopeo
-$ cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local
+Fedora$ cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local
+macOS$ cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local BUILDTAGS=containers_image_ostree_stub
 ```
```

Comment:
That depends on someone testing this and providing the content… Just in time, see #383 .

(Ideally, we would want the commands to be simple, and the same, everywhere. The manual `BUILDTAGS=` here is just an artefact of not atomating things enough.)